package.json

```diff
@@ -21,6 +21,26 @@
     "remark-lint": "~5",
     "stylelint": "~7",
     "stylint": "~1.5.9",
+    "textlint": "~7.3.0",
+    "textlint-plugin-asciidoc-loose": "~1.0.1",
+    "textlint-plugin-html": "~0.1.5",
+    "textlint-plugin-review": "~0.3.3",
+    "textlint-plugin-rst": "~0.1.1",
+    "textlint-rule-alex": "~1.2.0",
+    "textlint-rule-common-misspellings": "~1.0.1",
+    "textlint-rule-date-weekday-mismatch": "~1.0.5",
+    "textlint-rule-ginger": "~2.1.0",
+    "textlint-rule-max-comma": "~1.0.4",
+    "textlint-rule-max-number-of-lines": "~1.0.3",
+    "textlint-rule-ng-word": "~1.0.0",
+    "textlint-rule-no-dead-link": "~3.1.1",
+    "textlint-rule-no-empty-section": "~1.1.0",
+    "textlint-rule-no-start-duplicated-conjunction": "~1.1.3",
+    "textlint-rule-no-todo": "~2.0.0",
+    "textlint-rule-period-in-list-item": "~0.2.0",
+    "textlint-rule-rousseau": "~1.4.5",
+    "textlint-rule-unexpanded-acronym": "~1.2.1",
+    "textlint-rule-write-good": "~1.6.0",
```

Comment:
just curious: Our package requirements don't support the `~` operator, right? (yet)
to indicate a rough version specification^^